### DISCUSSION TITLE
Définition d’une classe de documents pour l’Encyclopédie

### DISCUSSION THREAD

**Discussion by emchateau** [ouvroir/encyclopedie#26] [General]
L’Encyclopédie nécessite l’utilisation d’un schéma qui définit une classe de document. Dans le projet l’utilisation de la TEI est nécessaire pour la pérennisation des fichiers mais ne doit pas être seulement une contrainte. On doit donc chercher à tirer partie des technologies XML pour assister les éditeurs dans la production des documents et assurer les contrôles nécessaires pour réduire les difficultés lors de la transformation des fichiers.

La définition d’un schéma pour l’Encyclopédie doit spécifier plusieurs types de document :
- entrées (placées dans un chapitre de l’Encyclopédie)
- encadrés (en lien avec un ou plusieurs articles, à l’intérieur d’un chapitre de l’Encyclopédie)
- compléments numériques (exposition, chronologie, living book) [les schémas seront à produire plus tard]

La conception du modèle peut s’inspirer du schéma TEI-Journal qui est très bien conçu pour assister l’édition des articles mais nécessite des adaptation au contexte francophone et pour les  besoins spécifiques de l’Encyclopédie. Il serait intéressant de pouvoir prendre connaissance des pratiques du schéma Métopes.

**Comment by pippaJeanne** [comment]
Pour l'odd, fera-t-on des schémas propres d'après les modèles qu'on a pris (TEI-Journal et Métopes) ? Est-ce qu'on aura plusieurs odds ? (L'un pour chaque type de documents, y compris les index avec <teiCorpus> ?)

**Comment by emchateau** [comment]
Cela peut nous aider de modulariser les modèle ODD pour la validation. On peut chaîner des ODD.
Mais j'aimerais que nous ayons un schéma global pour la validation du corpus.
